include/psa/crypto_values.h

```diff
@@ -2021,6 +2021,15 @@
 #define PSA_ALG_TLS12_PSK_TO_MS_GET_HASH(hkdf_alg)                         \
     (PSA_ALG_CATEGORY_HASH | ((hkdf_alg) & PSA_ALG_HASH_MASK))
 
+/* Macro to build a KDF that takes the shared secret K (an EC point in case
+ * of EC J-PAKE) and calculates SHA256(K.X) that the rest of TLS 1.2 will
+ * use to derive the session secret. Uses PSA_ALG_SHA_256. Only P-256 is
+ * supported, so the input has to be exactly 65 bytes.
```

Comment:
I'd turn the last sentence into its own paragraph, describing the input (like what's done for the other KDFs):
```
This function takes a single input:
* #PSA_KEY_DERIVATION_INPUT_SECRET is the shared secret K from EC J-PAKE. The only supported curve is secp256r1 (the 256-bit curve in #PSA_ECC_FAMILY_SECP_R1) so the input must be exactly 65 bytes.